### PR TITLE
NEPT-1068: Implement the new footer component.

### PR DESCRIPTION
**Pull request by voidtek**

### CODE REVIEW

File: templates/page/page--ds.tpl.php [hunk 1]
```diff
@@ -186,48 +186,6 @@
...
-      <div class="row">
-        <div class="col-sm-12">
-        <?php if (!empty($page['footer_bottom'])): ?>
-          <?php print render($page['footer_bottom']); ?>
-        <?php endif; ?>
-        </div>
-      </div>
-    </div>
-  </div>
+<footer class="ecl-footer">
```
Comment (drupol) [inline templates/page/page--ds.tpl.php]: I have a question regarding this... and to be honest, I don't know how to deal with the solution.
Should we include the `<footer>` tag inside the footer template or we should let it live in the page template ?

**Comment by netlooker** [comment]
There is an error (notice) in the current implementation. More details and screenshots in the JIRA ticket.

File: templates/page/page--ds.tpl.php (hunk 1 above)
Comment (voidtek) [inline templates/page/page--ds.tpl.php]: it's done.

**Comment by drupol** [comment]
👍

**Comment by gillesdeudon** [comment]
👍

File: templates/block/block--menu--menu-nexteuropa-inst-links.tpl.php [hunk 2]
```diff
@@ -46,7 +46,7 @@
 ?>
 <?php print render($title_prefix); ?>
 <?php if (!empty($title)): ?>
-  <h4 class="footer__title" <?php print render($title_attributes); ?>><?php print render($title); ?></h4>
+  <h4 class="ecl-h4 ecl-footer__title" <?php print render($title_attributes); ?>><?php print render($title); ?></h4>
```
Comment (voidtek) [inline templates/block/block--menu--menu-nexteuropa-inst-links.tpl.php]: We should not use both methods to insert classes (by the theme and by the preprocess)
the result is <h4 class="ecl-h4 ecl-footer__title"  class="block__title title">

File: templates/block/block--menu--menu-nexteuropa-site-links.tpl.php [hunk 3]
```diff
@@ -46,7 +46,7 @@
 ?>
 <?php print render($title_prefix); ?>
 <?php if (!empty($title)): ?>
-  <h4 class="footer__title" <?php print render($title_attributes); ?>><?php print render($title); ?></h4>
+  <h4 class="ecl-h4 ecl-footer__title" <?php print render($title_attributes); ?>><?php print render($title); ?></h4>
```
Comment (voidtek) [inline templates/block/block--menu--menu-nexteuropa-site-links.tpl.php]: We should not use both methods to insert classes (by the theme and by the preprocess)
the result is <h4 class="ecl-h4 ecl-footer__title"  class="block__title title">

File: templates/block/block--menu--menu-nexteuropa-social-media.tpl.php [hunk 4]
```diff
@@ -46,7 +46,7 @@
 ?>
 <?php print render($title_prefix); ?>
 <?php if (!empty($title)): ?>
-  <h4 class="footer__title" <?php print render($title_attributes); ?>><?php print render($title); ?></h4>
+  <h4 class="ecl-h4 ecl-footer__title" <?php print render($title_attributes); ?>><?php print render($title); ?></h4>
```
Comment (voidtek) [inline templates/block/block--menu--menu-nexteuropa-social-media.tpl.php]: We should not use both methods to insert classes (by the theme and by the preprocess)
the result is <h4 class="ecl-h4 ecl-footer__title"  class="block__title title">

File: templates/block/block--menu--menu-nexteuropa-site-links.tpl.php (hunk 3 above)
Comment (drupol) [inline templates/block/block--menu--menu-nexteuropa-site-links.tpl.php]: 👍

File: templates/block/block--menu--menu-nexteuropa-inst-links.tpl.php (hunk 2 above)
Comment (drupol) [inline templates/block/block--menu--menu-nexteuropa-inst-links.tpl.php]: 👍

File: templates/block/block--menu--menu-nexteuropa-social-media.tpl.php (hunk 4 above)
Comment (drupol) [inline templates/block/block--menu--menu-nexteuropa-social-media.tpl.php]: 👍

**Comment by netlooker** [comment]
👍

**Pull request merged**